Gems/AtomLyIntegration/CommonFeatures/Code/Source/Debug/RayTracingDebugComponentController.cpp

```diff
@@ -65,6 +71,11 @@ namespace AZ::Render
 
     void RayTracingDebugComponentController::Deactivate()
     {
+        if (IsRayTracingNotSupported())
+        {
+            return;
+        }
```

Comment:
While I understand that you had to add the `IsRayTracingNotSupported` method for the UI property, I think the readability of this code in these two methods would be better if it states `if(!IsRayTracingSupported())` as it's easier to miss the `Not` in the middle there than the `!` in the beginning. At least that happened to me as I reviewed the code just now.

Reply:
Changed to `if(!IsRayTracingSupported())` as suggested